ropsten-beacon-chain/config.yaml

```diff
@@ -0,0 +1,66 @@
+# Extends the mainnet preset
+PRESET_BASE: 'mainnet'
+CONFIG_NAME: 'ropstenbeaconchain'
```

Comment:
I would have expected this to just be `ropsten` given we're not giving the beacon chain a separate name. Otherwise we're going to forever be calling this reopstenbeaconchain.

Reply:
I would also expect this to be `ropsten`, given that we use `mainnet` not `mainnetbeaconchain`.

Reply:
I've renamed it to ropsten :D